plugins/versions.properties

```diff
@@ -30,3 +30,5 @@ version.okhttp3=4.8.1
 version.retrofit2=2.9.0
 
 version.moshi=1.9.3
+
+version.com.google.cloud..google-cloud-storage=1.113.2
```

Comment:
Run `$ ./gradlew refreshVersions` inside `plugins`, this way the file will be correctly sorted and you won't have a merge conflict